src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -71,6 +73,7 @@ public function __construct($name, $value = null, $expire = 0, $path = '/', $dom
         $this->secure = (bool) $secure;
         $this->httpOnly = (bool) $httpOnly;
         $this->raw = (bool) $raw;
+        $this->sameSite = $sameSite;
```

Comment:
What about validating the domain? I.e. allowed values (`"strict"/Cookie::SAMESITE_STRICT`, `"lax"/Cookie::SAMESITE_LAX` or `false`)